profiles/models.py

```diff
@@ -1,3 +1,447 @@
+        related_name='startups'
+    )
+    location = models.ForeignKey(
+        'Location',
+        on_delete=models.PROTECT,
+        related_name='startup_locations'
+    )
+    startup_logo = models.ImageField(upload_to='startups/logos/')
+    social_media = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        validators=[MaxLengthValidator(255)]
+    )
+    stage = models.CharField(
+        max_length=20,
+        choices=Stage.choices,
+        blank=True,
+        null=True
+    )
+
+    def __str__(self):
+        return f"{self.company_name} (Startup, User ID: {self.user_id})"
+
+    class Meta:
+        db_table = "startups"
+        ordering = ["company_name"]
+        verbose_name = "Startup"
+        verbose_name_plural = "Startups"
+        constraints = [
+            models.UniqueConstraint(fields=["user"], name="unique_startup_user")
+        ]
+
+
+class Investor(Company):
+    user = models.OneToOneField(
+        'users.User',
+        on_delete=models.CASCADE,
+        related_name='investor'
+    )
+    industry = models.ForeignKey(
+        'Industry',
+        on_delete=models.PROTECT,
+        related_name='investors'
+    )
+    location = models.ForeignKey(
+        'Location',
+        on_delete=models.PROTECT,
+        related_name='investor_locations'
+    )
+    investor_logo = models.ImageField(upload_to='investors/logos/')
+    interests = models.TextField(blank=True, null=True)
+    fund_size = models.DecimalField(
+        max_digits=20,
+        decimal_places=2,
+        blank=True,
+        null=True,
+        validators=[MinValueValidator(0)]
+    )
+    country = models.ForeignKey(
+        'Country',
+        on_delete=models.PROTECT,
+        related_name='investors'
+    )
+    preferred_stage = models.CharField(
+        max_length=20,
+        choices=Stage.choices,
+        blank=True,
+        null=True
+    )
+
+    def __str__(self):
+        return f"{self.company_name} (Investor, User ID: {self.user_id})"
+
+    class Meta:
+        db_table = "investors"
+        ordering = ["company_name"]
+        verbose_name = "Investor"
+        verbose_name_plural = "Investors"
+        constraints = [
+            models.UniqueConstraint(fields=["user"], name="unique_investor_user")
+        ]
+
+
+class StartupSocialMedia(models.Model):
+    startup = models.ForeignKey(
+        'Startup',
+        on_delete=models.CASCADE,
+        related_name='social_links'
+    )
+    platform = models.CharField(max_length=50)
+    url = models.URLField()
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.platform} for {self.startup.company_name}"
+
+    class Meta:
+        db_table = "startup_social_media"
+        ordering = ["-created_at"]
+        verbose_name = "Startup Social Media"
+        verbose_name_plural = "Startup Social Media Links"
+        unique_together = ("startup", "platform")
+
```

Comment:
Why not use a JSONField for social links as a dictionary (platform: URL)? It would simplify queries and avoid the need for a separate table unless fine-grained management is required.